app/views/supplementary/previous_documents.scala.html

```diff
@@ -28,15 +28,15 @@
     appConfig = appConfig
 ) {
     @helpers.form(PreviousDocumentsController.savePreviousDocuments(), 'autoComplete -> "off") {
-        @components.back_link("/customs-declare-exports/declaration/supplementary/package-information")
+        @components.back_link("/customs-declare-exports/declaration/supplementary/transaction-type")
 
         @components.error_summary(form.errors)
 
-        @components.page_title(Some("supplementary.previousDocuments.title"))
+        @components.page_title(Some("supplementary.previousDocuments.title"),Some("For example, 8GB-123456789101-SHIP1"))
```

Comment:
Please move `"For example, 8GB-123456789101-SHIP1"` to messages and put key here